index.html

```diff
@@ -1,174 +1,185 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="utf-8">
-    <title>Startrly</title>
-     <link href="styles/styles.css" rel="stylesheet">
+<head>
+  <meta charset="utf-8">
+  <title>Startrly</title>
+  <link href="styles/styles.css" rel="stylesheet">
 
-  </head>
-  <body>
-    <div>
-
-<!-- first box -->
-  <div class="intro">
-
-      <nav>
-        <ul>
-          <li>
-            Startrly
-          </li>
-          <li>
-            Try Now!
-          </li>
-          <li>
-            About
-          </li>
-        </ul>
-      </nav>
+</head>
+<body>
+  <div>
 
+    <!-- first wrapper start -->
+  <div class="wrapper_1">
```

Comment:
Hi Kat! First and foremost it was just great working with you. I really appreciated the time you took to purposefully design your project on paper before putting it in screen. Your level of organization will continue to support your journey as a software developer.